packages/angular-output-target/src/generate-proxies.ts

```diff
@@ -71,14 +71,15 @@ function getProxy(cmpMeta: ComponentCompilerMeta) {
     directiveOpts.push(`inputs: ['${inputs.join(`', '`)}']`);
   }
   if (outputs.length > 0) {
-    directiveOpts.push(`outputs: ['${outputs.join(`', '`)}']`)
+    directiveOpts.push(`outputs: ['${outputs.map(output => output.name).join(`', '`)}']`)
   }
 
   const tagNameAsPascal = dashToPascalCase(cmpMeta.tagName);
 
+  const typePath = path.relative(componentCorePackage, cmpMeta.sourceFilePath).replace('../src', path.join(componentCorePackage, distTypesDir)).replace('.tsx', '');
   const outputsInterface =
   outputs.length > 0
-    ? `export declare interface ${tagNameAsPascal} extends Required<Pick<JSX.${tagNameAsPascal}, '${outputs.map(o => 'on' + o.replace(/^./, match => match.toUpperCase())).join(`' | '`).replace(/\| $/g, '')}'>> {}`
+    ? `import { ${cmpMeta.componentClassName} as I${cmpMeta.componentClassName} } from '${typePath}';`
```

Comment:
I guess this makes much more sense - however, build is failing due to unresolved types that exist in the global scope of the component-library (i.e. packages/example-project/component-library/src/interfaces.d.ts). Suggestions?